pkgs/jnigen/lib/src/elements/elements.dart

```diff
@@ -33,13 +33,9 @@ class Classes implements Element<Classes> {
 
   final Map<String, ClassDecl> decls;
 
-  factory Classes.fromJson(List<dynamic> json) {
-    final decls = <String, ClassDecl>{};
-    for (final declJson in json) {
-      final classDecl = ClassDecl.fromJson(declJson);
-      decls[classDecl.binaryName] = classDecl;
-    }
-    return Classes(decls);
+  factory Classes.fromJson(Map<String, dynamic> json) {
```

Comment:
Apparently this is not covered with tests according to the coverage bot?

Reply:
It should be, we don't have coverage link anymore? We used to get them from codecov bot in jnigen repo. @mosuem Could we set that up here as well?

Reply:
Nevermind, the tests were not all passed before.